packages/mobile/src/icons/CircleArrowIcon.tsx

```diff
@@ -0,0 +1,38 @@
+import { elevationShadowStyle } from '@celo/react-components/styles/styles'
+import * as React from 'react'
+import { StyleSheet, View, ViewStyle } from 'react-native'
+import Svg, { Path } from 'react-native-svg'
+
+const SIZE = 24
+
+interface Props {
+  style?: ViewStyle
+}
+
+export default function CircleArrowIcon({ style }: Props) {
+  return (
+    <View style={[styles.container, style]}>
+      <Svg width={13} height={9} viewBox="0 0 13 9" fill="none">
+        <Path
+          d="M1 4.5h11m0 0L8.264 1M12 4.5L8.264 8"
+          stroke="#2E3338"
```

Comment:
nit: If this matches an existing color we have consider using that color variable here